### PR TITLE
Use stable mypy

### PR DESCRIPTION
**Pull request by srittau**
Pin all package versions

### CODE REVIEW

**Comment by JelleZijlstra** [comment]
I think the stubtest and/or the mypy-primer flow has a hardcoded mypy commit hash somewhere. Could you update that too?

**Comment by srittau** [comment]
Done.

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -56,8 +56,9 @@ jobs:
       - uses: actions/setup-python@v2
         with:
           python-version: 3.9
-      - run: pip install -r requirements-tests-py3.txt
-      - run: ./tests/pytype_test.py
+      - run: |
+          pip install -r requirements-tests-py3.txt
+          ./tests/pytype_test.py
```
Comment (Akuli) [inline .github/workflows/tests.yml:61]: I would prefer these as separate steps. That way it's easier to see which command failed. This also applies to `mypy:` job.

File: .github/workflows/stubtest-unused-whitelist.yml [hunk 2]
```diff
@@ -25,11 +25,8 @@ jobs:
         uses: actions/setup-python@v2
         with:
           python-version: ${{ matrix.python-version }}
-      # Keep the mypy version in sync with what we use for stubtest in tests.yml
       - name: Install dependencies
-        run: |
-          python -m pip install -U pip
-          pip install -U git+git://github.com/python/mypy@master
+        run: pip install -U $(grep mypy== requirements-tests-py3.txt)
```
Comment (Akuli) [inline .github/workflows/stubtest-unused-whitelist.yml:29]: Nit: `-U`/`--upgrade` is not necessary, since we specify the exact version to install.

**Review by Akuli — APPROVED** [review]

**Pull request merged**